charts/pega/templates/_helpers.tpl

```diff
@@ -13,11 +13,19 @@
 
 {{- define "pegaImportCertificatesTemplate" }}
 - name: {{ template "pegaVolumeImportCertificates" }}
-  configMap:
-    # This name will be referred in the volume mounts kind.
-    name: {{ template "pegaImportCertificatesConfig" $ }}
-    # Used to specify permissions on files within the volume.
+  projected:
     defaultMode: 420
+    sources:
+  {{ if (.Values.global.certificatesSecret) }}
+  {{- range .Values.global.certificatesSecret }}
+    - secret:
+        name: {{ . }}
+  {{- end }}
+  {{ else }}
+    # This name will be referred in the volume mounts kind.
+    - secret:
+        name: {{ template "pegaImportCertificatesConfig" $ }}
```

Comment:
I recommend merging `charts/pega/templates/_pega-certificates-config.tpl` code to this place, this way multiple conditions can be avoided and `_pega-certificates-config.tpl` can be removed altogether. 
Or do vice-versa

Reply:
Discussed with @kishorv10  on this , hence resolving